.github/workflows/gin.yml

```diff
@@ -9,6 +9,20 @@ on:
       - master
 
 jobs:
+  golangci-lint:
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    steps:
+      - name: Setup go
+        uses: actions/setup-go@v2
+        with:
+          go-version: '^1.16'
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Setup golangci-lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: latest
```

Comment:
Don't use the `latest` version. Please define the specific version `v1.41.1`.

```yml
      with:
        args: --verbose
        version: v1.41.1
```

Reply:
```suggestion
          version: v1.41.1
          args: --verbose
```